Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py

```diff
@@ -203,6 +207,13 @@ def main():
     timeout = params.get('timeout')
     retries = params.get('retries') or 5
     aws_queue_url = params.get('queueUrl')
+    max_fetch_param = params.get('max_fetch', 10)
+    if max_fetch_param < 1:
+        max_fetch = 1
+    elif max_fetch_param > 100:
+        max_fetch = 100
+    else:
+        max_fetch = max_fetch_param
```

Comment:
```suggestion
    max_fetch = min(params.get('max_fetch', 10), 100)
```

Reply:
The problem with this is there needs to be a hard cap of 1 on the `max_fetch` which this dropped. To do what you're suggesting here we would need:
```python
max_fetch = max(min(params.get('max_fetch', 10), 100), 1)
```
This prevents a `max_fetch` of zero or negative numbers which would cause issues in the SQS API call which now directly uses the parameter. But when put into a single line it's not very readable or clear as to what's going on.

Reply:
i get what you're trying to say, but it does not make any sense to pass a value lower than 1, i dont think anyone  would pass -1 for example as a max fetch, in case they did, then its fine to get an error related to it.
the edit i added is good.
thanks for the effort tho!